doc/howto/network_ovn_peers.md

```diff
@@ -5,34 +5,56 @@ discourse: 12165
 (network-ovn-peers)=
 # How to create peer routing relationships
 
-Network peers allow the creation of routing relationships between two OVN networks.
-This allows for traffic between those two networks to stay within the OVN subsystem rather than having to transit via the uplink network.
+By default, traffic between two OVN networks goes through the uplink network.
+This routing is inefficient, however, because it limits the network bandwidth to the uplink network's capabilities.
 
+Therefore, LXD allows creating peer routing relationships between two OVN networks.
+Using this method, traffic between the two networks can go directly from one OVN network to the other and thus stays withing the OVN subsystem, rather than transiting through the uplink network.
 
 ## Create a routing relationship between networks
 
-Both networks in the peering are required to complete a setup step to ensure that the peering is mutual.
+To add a peer routing relationship between two networks, you must create a network peering for both networks.
+Use the same peering name for both networks to ensure that the relationship is mutual.
 
-E.g.
+Use the following commands to create a peer routing relationship:
 
-```
-lxc network peer create <local_network> foo <target_project/target_network> --project=local_network
-lxc network peer create <target_network> foo <local_project/local_network> --project=target_project
+    lxc network peer create <network1> <peering_name> <network2> [configuration_options]
+    lxc network peer create <network2> <peering_name> <network1> [configuration_options]
+
+You can also create peer routing relationships between OVN networks in different projects:
+
+    lxc network peer create <network1> <peering_name> <project2/network2> [configuration_options] --project=<project1>
+    lxc network peer create <network2> <peering_name> <project1/network1> [configuration_options] --project=<project2>
+
+```{important}
+If the project or the network name is incorrect, the command will not return any error indicating that the respective project/network does not exist.
```

Comment:
Would be to good to say explicitly that the network relationship will be left in pending status in these scenarios.